frontend/projects/ui/src/app/components/toast-container/refresh-alert/refresh-alert.component.ts

```diff
@@ -10,13 +11,23 @@ import { RefreshAlertService } from './refresh-alert.service'
 })
 export class RefreshAlertComponent {
   private readonly dismiss$ = new Subject<boolean>()
-
   readonly show$ = merge(this.dismiss$, this.refresh$)
 
   constructor(
     @Inject(RefreshAlertService) private readonly refresh$: Observable<boolean>,
+    private readonly updates: SwUpdate,
   ) {}
 
+  async pwaReload() {
```

Comment:
Does this need a loading spinner? How long might it take?

Reply:
Uncertain, need to test it live further. It is probably a good idea to have one. Will add.